app/form_models/jobseekers/account_feedback_form.rb

```diff
@@ -0,0 +1,8 @@
+class Jobseekers::AccountFeedbackForm
+  include ActiveModel::Model
+
+  attr_accessor :origin, :comment, :feedback_type, :jobseeker_id, :rating
+
+  validates :comment, length: { maximum: 1200 }, if: -> { comment.present? }
```

Comment:
I think you can just do:
```
validates :comment, length: { maximum: 1200 }, if: :comment?
```

Reply:
The form doesn't have a `comment?` method - that is a method available in models.